pkg/cmd/release/create/http.go

```diff
@@ -189,12 +189,16 @@ func createRelease(httpClient *http.Client, repo ghrepo.Interface, params map[st
 	return &newRelease, err
 }
 
-func publishRelease(httpClient *http.Client, releaseURL string, discussionCategory string) (*shared.Release, error) {
+func publishRelease(httpClient *http.Client, releaseURL string, discussionCategory string, isLatest *bool) (*shared.Release, error) {
```

Comment:
@samcoe : should we consider [reusing the parameter preparation for creating the release](https://github.com/cli/cli/blob/7738b6187c80f128728aeba15fdba6bf62ec9e1b/pkg/cmd/release/create/create.go#L407-L427) since most of the parameters are reused for updating a release?

Reply:
That's a good question. At the moment I don't think it is worth making the extra effort to extract that logic and make it re-usble. I don't have strong opinions though.